.github/workflows/rust.yml

```diff
@@ -44,3 +42,31 @@ jobs:
       run: cargo clippy --all --all-features -- -D warnings
     - name: Format
       run: cargo fmt --check --verbose
+
+  test_release:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: ⚡ Cache rust
+      uses: actions/cache@v3
+      with:
+        path: |
+          ~/.cargo/registry
+          ~/.cargo/git
+          target
+        key: ${{ runner.os }}-cargo-release-${{ hashFiles('**/Cargo.toml') }}
+    # we need nightly for the `-Z compile-std=core` option for riscv
+    - name: Install nightly cargo
+      run: rustup default nightly
```

Comment:
we don't need this for powdr itself, only for the crates we wanna compile to riscv